xml/pop_banners.xml

```diff
@@ -238,7 +238,15 @@ matched against these patterns to fingerprint POP3 servers.
       <param pos="0" name="os.device" value="General"/>
       <param pos="1" name="os.version"/>
       <param pos="2" name="host.name"/>
-   </fingerprint>
+    </fingerprint>
+
+    <fingerprint pattern="^Hello there\.$">
```

Comment:
How confident are you that this is exclusively (or predominantly) used by Courier? Seems like a fairly generic greeting.